es-es/asciidoc-es.html.markdown

```diff
@@ -0,0 +1,138 @@
+---
+language: asciidoc
+contributors:
+    - ["Ryan Mavilia", "http://unoriginality.rocks/"]
+    - ["Abel Salgado Romero", "https://twitter.com/abelsromero"]
+lang: es-es
+filename: asciidoc-es.md
+---
+
+AsciiDoc es un lenguaje de marcas similar a Markdown que puede ser usado para cualquier uso, desde libros a blogs.
+Creado en 2002 por Stuart Rackham, el lenguaje es simple pero permite un gran nivel de personalización.
+
+Cabecera de documento
+
+La cabecera es opcional y no puede contener lineas vacías. Debe estar separada del contenido por al menos una línea en blanco.
+
+Solo título
+
+```
+= Título de documento
+
+Primer contenido del documento.
+```
+
+Título y autor
+
+```
+= Título del documento
+Nombre Apellido(s) <nombre.apellido@learnxinyminutes.com>
+
+Inicio de este documento.
+```
+
+Múltiples autores
+
+```
+= Título del documento
+John Doe <john@go.com>; Jane Doe<jane@yo.com>; Black Beard <beardy@pirate.com>
+
+Inicio de un documento con múltiples autores.
+```
+
+Linea de versión (requiere línea de autor)
+
+```
+= Titulo V1
+Potato Man <chip@crunchy.com>
+v1.0, 2016-01-13
+
+Este artículo sobre patatas fritas será genial.
+```
+
+Párrafo
+
+```
+No necesitas nada especial para un párrafo.
+
+Inserta una línea vacía entre cada párrafo para separarlos.
+
+Para insertar un salto de línea, solo añade un +
+y ya lo tienes!
+```
+
+Dando formato al texto
+
+```
+_guión bajo para cursiva_
+*asteriscos para negrita*
+*_combinalos y verás_*
+`usa comillas invertidas para monospace`
```

Comment:
Everyone I know uses monospace, probably because it's typical of technical environments.
I know the term "monoespaciado", but sounds weird. Dunno, whatever you feel more comfortable.